src/Blumchen.DependencyInjection/Blumchen.DependencyInjection.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
**FIX:** Let's merge it into the regular project. I'm fine with having hosting.abstractions and Polly dependency in the main project. It's much more accessible when user installs just a package and have all available (of course, unless that's enforcing too much from the peer dependency side).

Reply:
I've no strong opinion against it

Reply:
@lsfera, could you change that in the PR?